src/cms/hooks/useDisableWheel.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from "react";
+
+const useDisableWheel = () => {
+    useEffect(() => {
+        document.addEventListener("wheel", function (event) {
+            if (
+                document.activeElement &&
+                (document.activeElement as HTMLInputElement).type === "number"
+            ) {
+                (document.activeElement as HTMLInputElement).blur();
+            }
+        });
+    }, []);
+};
```

Comment:
Will this also blur on laptop touchpad scrolling? Is that behavior jarring if a user is in the process of editing an input field and scrolls slightly?

Reply:
Blurring on any scroll event seems kind of heavy-handed as a solution. Is it possible to just disable the handlers on the numeric input fields? Is there a configuration option in the CMS that lets you change this?

Reply:
unfortunately no, we haven't found one. Right now this is causing more issues than the connivence of scrolling helps. Your first question sounds like the issue: the user was trying to scroll down the page and accidentally changed the number value and didn't notice

Reply:
Yeah this hook targets only the number fields in the CMS admin page, and it also works on touchpads. The goal of this approach is to prevent accidental changes to numeric values. I've checked out decap cms options for number widget, but it currently doesn't have a option to handle this.